src/site/_collections/posts-with-lighthouse.js

```diff
@@ -21,7 +21,7 @@ const livePosts = require('../_filters/live-posts');
 
 module.exports = (collection) => {
   return collection
-    .getFilteredByTag('pathItem')
+    .getFilteredByTag('post')
```

Comment:
Is this a restriction in the number of posts that will appear in this collection?

IIRC from our conversation, `pathItem` is everything, and `post` is just the blog posts.

Reply:
The idea is to change our tags that "everyting" is `post` and blog posts are `blog`. So `s/pathItem/post `and `s/post/blog`.

Reply:
If we get rid of `pathItem` and blog posts get a `blog` tag added to them, then do we need a `post` tag at all? Could we instead do `collection.getFilteredByGlob("**/*.md")` to get all of the posts?

Also, a tiny optimization might be to move the filter that happens on line 26 so that it happens before the filter that happens on line 25. Then we'd be sending a much smaller subset of posts through `livePosts`.

This actually brings up a bigger point which is that I think we do `filter(livePosts)` in a number of places, and it might make sense to just do that once, and then have other collections use that data. But we can worry about that in a separate PR.

Reply:
We need "post" tag to distinguish it from the other pages, e.g. landing pages.

To me livePosts seems less expensive than filter in line 26, so it is better to perform it first (less items have to go through filter in 26). If we feel a need to optimize it (do we? this call happens only at build time) we should combine both filters to do only one  pass through posts, e.g.:

```
module.exports = (collection) => {
  return collection
    .getFilteredByTag('post')
    .filter((post) => {
      if (!livePosts(post)) {
         return false;
      }
      const audits = post.data.web_lighthouse;
      if (typeof audits === 'string' && audits !== 'N/A') {
        return true;
      }
      return audits instanceof Array && audits.length;
    });
};
``` 
IMO a bit less readable though. Let me know wdyt.

Reply:
I like your suggestion. I do like to optimize these loops because if you're writing an article on the site, it tries to rebuild every time you hit save. It used to do so pretty quickly but we've slowly been increasing that time (both by adding more docs and adding more collections that loop over those docs). Although maybe the collection looping isn't the real bottleneck (it may just take a long time to write the files to disk)